scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.7.1] - 2017-11-07
+### Added
+- Added support for killing a group of jobs, from @DaoWen
+- Added sysouts to get job output closer to Mesos' CommandExecutor, from @shamsimam
+- Added metrics support for usage of /list, from @dposada
+- Added support for groups on retry endpoint, from @DaoWen
+- Added support for configurable progress file in Cook Executor, from @shamsimam
+- Added support for configurable environment passed to Cook Executor, from @shamsimam
+### Changed
+- Fixed bug with job group constraints, from @pschorf
+- Fixed bug where Cook Executor jobs were opting in to the heartbeat support, from @shamsimam
+- Changed (simplified) the sandbox directory syncing mechanism for jobs, from @shamsimam
+- Removed jobs from group entity in job-ent->map, from @pschorf
```

Comment:
Did the previous release actually have this issue? I thought it was introduced in an unreleased change

Reply:
Spoke with @pschorf, removed this entry.